File: bookManageFrontDev/src/model/Flowrule.js
/**@param id:class java.lang.Integer
@param name:class java.lang.String
@param flowpattern:class java.lang.String
@param thresholdtype:class java.lang.String
@param threshold:class java.lang.Integer
@param thresholdpattern:class java.lang.String
@param floweffect:class java.lang.String
@param requestsource:class java.lang.String
@param status:class java.lang.Boolean
@param time:class java.util.Date
*/
export default class Flowrule {
    constructor() {
        this.id = ''
        this.name = ''
        this.flowpattern = ''
        this.thresholdtype = ''
        this.threshold = ''
        this.thresholdpattern = ''
        this.floweffect = ''
        this.requestsource = ''
        this.status = ''
        this.time = ''
    }
}

File: bookManageBackDev/src/main/java/com/Httb/bookManage/mbg/entity/Flowrule.java
package com.Httb.bookManage.mbg.entity;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import java.io.Serializable;
import java.util.Date;
import lombok.Data;

/**
 * flowrule
 * @author 
 */
@ApiModel(value="com.Httb.bookManage.mbg.entity.Flowrule")
@Data
public class Flowrule implements Serializable {
    /**
     * 流控id
     */
    @ApiModelProperty(value="流控id")
    private Integer id;

    /**
     * 规则名称
     */
    @ApiModelProperty(value="规则名称")
    private String name;

    /**
     * 流控模式
     */
    @ApiModelProperty(value="流控模式")
    private String flowpattern;

    /**
     * 阈值类型
     */
    @ApiModelProperty(value="阈值类型")
    private String thresholdtype;

    /**
     * 阈值
     */
    @ApiModelProperty(value="阈值")
    private Integer threshold;

    /**
     * 阈值模式
     */
    @ApiModelProperty(value="阈值模式")
    private String thresholdpattern;

    /**
     * 流控效果
     */
    @ApiModelProperty(value="流控效果")
    private String floweffect;

    /**
     * 请求来源
     */
    @ApiModelProperty(value="请求来源")
    private String requestsource;

    /**
     * 流控是否删除
     */
    @ApiModelProperty(value="流控是否删除")
    private Boolean status;

    /**
     * 流控创建时间
     */
    @ApiModelProperty(value="流控创建时间")
    private Date time;

    private static final long serialVersionUID = 1L;
}